### PR TITLE
feat : Implement breadcrumb portlet - EXO-66554 - Meeds-io/MIPs#87

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

File: webapp/portlet/src/main/webapp/WEB-INF/portlet.xml [hunk 1]
```diff
@@ -903,4 +903,22 @@
       <keywords>Platform Settings</keywords>
     </portlet-info>
   </portlet>
+
+  <portlet>
+    <portlet-name>Breadcrumb</portlet-name>
+    <display-name>Breadcrumb Portlet for site </display-name>
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/WEB-INF/portlet.xml]:
```suggestion
    <display-name>Site Breadcrumb</display-name>
```

File: webapp/portlet/src/main/webapp/WEB-INF/jsp/portlet/breadcrumb.jsp [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<%@ page import="org.exoplatform.portal.mop.user.UserNode" %>
+<%@ page import="org.exoplatform.portal.webui.util.Util" %>
+<%
+  UserNode selectedUserNode = Util.getUIPortal().getSelectedUserNode();
+  String nodeId = selectedUserNode.getId();
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/WEB-INF/jsp/portlet/breadcrumb.jsp]:
```suggestion
  UserNode selectedNode = Util.getUIPortal().getSelectedUserNode();
```

File: webapp/portlet/src/main/webapp/WEB-INF/jsp/portlet/breadcrumb.jsp [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+%>
+<div class="VuetifyApp">
+  <div id="breadcrumb">
+    <script type="text/javascript">
+      eXo.env.portal.selectedNodeId = '<%= nodeId %>'
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/WEB-INF/jsp/portlet/breadcrumb.jsp]:
```suggestion
      eXo.env.portal.selectedNodeId = '<%= selectedNode.getId()%>'
```

**Review by azayati — CHANGES_REQUESTED** [review]

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**